composer.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "sylius/calendar",
   "type": "library",
-  "version": "v1.0-dev",
+  "version": "v0.0.1-dev",
```

Comment:
I wonder if it shouldn't be `"v1.0-dev"` or at least `"v0.1-dev"`

Reply:
`v0.1-dev` indeed 👍